### DISCUSSION TITLE
Thank you for sharing nice models!

### DISCUSSION THREAD

**Discussion by sanbuphy** [OpenModelDB/open-model-database#274] [General]
Thank you for sharing so many excellent models! I have learned a lot. Could you please provide some methods for training ESRGAN, ResRGAN, or removing artifacts? I would greatly appreciate it. Thank you very much for your efforts.

**Comment by joeyballentine** [comment]
Thanks for the kind words. Here is a tutorial on YouTube for how to get started with training: https://www.youtube.com/watch?v=iH7-eYlf7eg

**Comment by sanbuphy** [comment]
> Thanks for the kind words. Here is a tutorial on YouTube for how to get started with training: https://www.youtube.com/watch?v=iH7-eYlf7eg

Thanks you for the cool video! but I not sure if i can run the traning because i just have the NVIDIA3060, may be it will take so much time.
